src/environment/standalone/StandAloneEnvironment.cpp

```diff
@@ -120,12 +121,41 @@ void StandAloneEnvironment::reloadMetar() {
     xplaneData->reloadMetar();
 }
 
-Location StandAloneEnvironment::getAircraftLocation() {
-    Location res{};
-    res.latitude = 53.8019434;
-    res.longitude = 10.7017287;
-    res.heading = 70;
-    return res;
+unsigned int StandAloneEnvironment::getActiveAircraftCount() {
+    return 4;
+}
+
+Location StandAloneEnvironment::getAircraftLocation(unsigned int id) {

```

Comment:
This is cool, I like the idea of having moving AI aircraft in the standalone version for testing!